src/loaders/FileLoader.js

```diff
@@ -19,7 +30,10 @@ class FileLoader extends Loader {
 
 		url = this.manager.resolveURL( url );
 
-		const cached = Cache.get( url );
+		const isRangeRequest = this.requestHeader.Range !== undefined;
```

Comment:
Any reason for range header to be special? Might want to just include all headers in cache key.

Reply:
Because we Three.js have never heard problems for other headers so far. Perhaps we may rethink if we get problem report.